dictionaries/html-symbol-entities/entities.txt

```diff
@@ -1,25 +1,51 @@
+&Aacute;
```

Comment:
Should the leading/trailing punctuation be stripped off?

Reply:
Yes, they should be removed.

The tool to build the dictionary can strip them off, but I think it is better to just remove them.